include/swift/AST/SearchPathOptions.h

```diff
@@ -504,6 +504,10 @@ class SearchPathOptions {
   /// version inheritance.
   std::optional<std::string> PlatformAvailabilityInheritanceMapPath;
 
+  /// Cross import module information.
+  using CrossImportMap = llvm::StringMap<std::vector<std::string>>;
```

Comment:
Can we mention what the format of the data is? stringmap<vector<string>> is not obvious what is going on